### DISCUSSION TITLE
Assignment manually on event

### DISCUSSION THREAD

**Discussion by tiicaa** [liedekef/events-made-easy#149] [Q&A]
Is it possible to assign participation in an event manually? For example, a user reports that they are unable to register for a certain event. Would I be able to assign this event to this user who already has a registration via the panel?
What I tried was me as an administrator registering for events and then assigning them to the registered user. Is this the way or is there no other way?

**Comment by liedekef** [comment]
Click in the backend on "RSVP" for the event of choice in the list of events.

**Reply by tiicaa** [reply]
Thanks for the feedback
